gf-webapp/src/component/pageBorder/header/contacts/contactsList/ContactsList.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from "src/component/pageBorder/header/contacts/contactsList/ContactsList.module.scss";
+
+interface Contact {
+  type: string;
+  contact: string;
+}
+
+const contactsList: Contact[] = [
+  {
+    type: "tel:",
```

Comment:
Better to rename it to contactType or something similar

Reply:
Why? Const include contacts in naming so property is clear that it is type of contacts. 
It's the similar as in our model - room had property type. We did not call ir roomType